service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AmendedReturnedCaseEventHandler.java

```diff
@@ -27,10 +30,22 @@ public void notifyAdmin(AmendedReturnedCaseEvent event) {
 
         final String email = adminNotificationHandler.getHmctsAdminEmail(eventData);
 
-        ReturnedCaseTemplate parameters = returnedCaseContentProvider
-            .buildNotificationParameters(eventData.getCaseDetails(), eventData.getLocalAuthorityCode());
+        ReturnedCaseTemplate parameters = contentProvider.parametersWithCaseUrl(eventData.getCaseDetails());
 
-        notificationService.sendEmail(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN, email, parameters,
+        notificationService.sendEmail(AMENDED_APPLICATION_RETURNED_ADMIN_TEMPLATE, email, parameters,
+            eventData.getReference());
+    }
+
+    @Async
```

Comment:
I  like the async Usage, makes it better for the user.

Do you know how big by default is that async thread pool?

Reply:
In AsyncConfigrattion we deined to use
ThreadPoolTaskExecutor
which have

public class ThreadPoolTaskExecutor extends ExecutorConfigurationSupport
		implements AsyncListenableTaskExecutor, SchedulingTaskExecutor {

	private final Object poolSizeMonitor = new Object();

	private int corePoolSize = 1;

	private int maxPoolSize = Integer.MAX_VALUE;